client/src/pages/ForumPage/ForumPage.js

```diff
@@ -166,7 +168,7 @@ const ForumPage = props => {
         </div>
         <div>
           <header className="ForumPage__header">
-            #hashtags I'm Following: <a href="/forum-page-hashtag">Example Tag</a>
+            #hashtags I'm Following: <a href="/forum-page-hashtag/1">Example Tag</a>
```

Comment:
This too? I think @Bcastillo44  is working on a PR related to having the pills actually work

Reply:
@Bcastillo44  will have a whole new page.  I was just updating the placeholder.